test-cypress/integration/home/workbench.home.spec.js

```diff
@@ -42,6 +42,9 @@ describe('Home screen validation', function () {
         HomeSteps.verifyCreateRepositoryLink();
 
         let repositoryId = HomeSteps.createRepo();
+        // Warming the namespaces to speed up retrieving repository info
+        cy.warmRepositoryNamespaces(repositoryId);
```

Comment:
The delay is because the repository is not initialized. It gets initialized the first time a request to the repo is sent (get namespaces, execute query, get repo size). I suggest to move this to createRepo and send a GET /repositories/{repositoryID}/size to initialize it after creation since get size is the simplest. You can also leave it with the call to get namespaces, but please, rename the function to initialize repository.